cmd/weaviate/main.go

```diff
@@ -58,12 +60,22 @@ func main() {
 		// TODO(kavi): Find a way to share this functionality in both go-client and in querytenant.
 		tenantInfo := querytenant.NewTenantInfo(opts.Query.SchemaAddr, querytenant.DefaultSchemaPath)
 
+		vclient, err := client.NewClient(opts.Query.VectorizerAddr, log)
+		if err != nil {
+			log.WithFields(logrus.Fields{
+				"cause": err,
```

Comment:
let's use `err` instead of `cause` to be consistent with what we are doing everywhere else in the code

Reply:
And you can also use `.Err(err)` to add a field to log the error directly in the logger :)

Reply:
Done.